presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpcds.yaml

```diff
@@ -2,7 +2,7 @@ datasource: presto
 query-names: presto/tpcds/${query}.sql
```

Comment:
I do not get the commit message.

`reorder_joins is not true by default,`, so it is `false` by default. 

`therefore it doesn't makes sense to benchmark legacy reorder_joins=false setting.` Why? It does not matter what default value is if you test both of them?

Reply:
Basically since `reorder_joins=true` by default in presto, we can remove benchmarking of legacy `reorder_joins=false` because it doesn't make much sense to benchmark queries with cross join

Reply:
> because it doesn't make much sense to benchmark queries with cross join

It depends.

Reply:
@kokosing we've checked that in all TPCH/DS queries =false doesn't degrade performance. I think we can stop running benchmarks with =false